server/service/store/service.go

```diff
@@ -0,0 +1,11 @@
+package store
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func RegisterHandlers(router *mux.Router) {
+	router.HandleFunc("/v1/data/{key}", HandleStoreData).Methods(http.MethodPost)
```

Comment:
Why don't we add `deal_id`? To download their data by deal?

Reply:
The `key` which means `datahash` is unique so that I think it is not need to add `deal_id`. But if we add a `deal_id` it could be check that the `datahash` is in valid deal.

Reply:
Actually, `dataHash` is unique in a `deal`. The same data (same `dataHash`) can be provided to several deals If several deals desire the same type of data.

Reply:
And I don't think there is no need to check the validity of data. It is guaranteed by oracle.
But I'll think about it more just in case, because we have some changes these days.

Reply:
> But if we add a deal_id it could be check that the datahash is in valid deal.
For this, I think it needs to add `deal_id` in url.

Reply:
Applied the `deal_id` in [d747c40](https://github.com/medibloc/panacea-dep-consumer/pull/1/commits/d747c40186a8a3637f5c184c3a8be1601eff5eb2).